### ISSUE TITLE
When Screen Reader is ON, Radzen Grid rows cannot be accessed using Arrow Keys

### ISSUE THREAD

**Issue by truecaller12** [radzenhq/radzen-blazor#2530]
**Describe the bug**
When **Screen Reader is ON**, Radzen Grid rows cannot be accessed using Arrow Keys. It works without Screen reader

**To Reproduce**
1. Go to 'https://blazor.radzen.com/datagrid'
2. Tab to reach the Data Grid control  and use Arrow keys to select rows
3. Rows cannot be selected

**Expected behavior**
With Screen reader, you should be able to navigate through the rows with arrow keys

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Edge

**Comment by enchev** [comment]
Fixed in 1e184ff52.

**Comment by truecaller12** [comment]
Upgraded to 10.3.1, the issue is reproducible.

**Comment by truecaller12** [comment]
Upgraded to 10.3.2, the issue is reproducible.

**Comment by enchev** [comment]
Unfortunately I’m out of other ideas/fixes. Feel free to submit pull request if know how this can be resolved.

**Comment by enchev** [comment]
Fixed in 1e184ff52 and 6f5b0285f. The DataGrid now uses role="grid"/treegrid with aria-activedescendant on the focusable wrapper, so screen readers switch to focus mode and arrow-key row navigation works. Update to the latest release.
